docs/organizations/managing-security-and-risk.md

```diff
@@ -0,0 +1,108 @@
+# Managing security and risk
+
+!!! info "This is a beta feature"
+    This is a new Codacy feature and <span class="skip-vale">we're</span> continuing to improve it.
+
+To <span class="skip-vale">quickly</span> identify, prioritize, track, and address security vulnerabilities, open the **Security and risk management** page from the organization-level navigation menu. On this page, you can monitor and act on time-bound, prioritized action items that Codacy creates automatically when finding Security issues:
+
+-   Codacy opens a new item when it finds a new Security issue on the default branch of a repository.
+
+    [Every new item is assigned a priority](#item-priorities-and-deadlines) based on the severity of the underlying issue. Each priority also defines a deadline to fix the underlying issue.
+
+    [Every new item is assigned a status](#item-statuses) based on the number of days left before the deadline to fix the underlying issue. As deadlines are approached, met, or missed, items transition through different statuses.
+
+-   Codacy closes an item when it stops detecting the underlying issue, when [the underlying issue is ignored](../repositories/issues.md#ignoring-and-managing-issues), or when the [tool that found the underlying issue is disabled](../repositories-configure/configuring-code-patterns.md).
+
+For example, Codacy automatically opens an item with Critical priority upon finding a new Security issue of Critical severity on the default branch of a repository. If the Security issue is fixed before the deadline, the status of the item changes to Closed on time. If the issue isn't fixed before the deadline, the status of the item changes to Overdue.
```

Comment:
All these details about Codacy automation could be explained later on this page. I feel that we should describe the Dashboard areas first, and these details only after (incremental information disclosure).

Reply:
Perhaps the details about how Codacy maps the items to a priority and status can be explained only inside their corresponding sections, thus making this part more lightweight and easier to understand on a first read.

Anyway, this is bound to keep evolving so we'll need to perform some refactoring later on either way.

Reply:
Yes, I'll keep it simple and just move this to the "Opening and closing items" section for now.